src/style.css

```diff
@@ -0,0 +1,90 @@
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 0;
+  display: flex;
+  width: 96px;
+  height: 96px;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  align-self: auto;
+
+}
+
+.stars__star {
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--0 .stars__star {
```

Comment:
For the stars--0 class, you should use the inactive star image instead of the active star image.